server/handler/status.go

```diff
@@ -137,6 +138,9 @@ func (h *Status) processOthers(ctx context.Context, event github.StatusEvent) er
 		})
 		if err != nil {
 			evaluationFailures++
+			logger.Error().Err(err).Msgf("failed to evaluate pull request '%d' for SHA '%s'", pr.GetNumber(),
```

Comment:
style: `Failed to ...`. In this codebase, log message usually start with a captial letter, errors with a lowercase letter.